### PR TITLE
Fix Travis

### LINKED ISSUE #12: Travis build is broken, so the GitHub Pages site is not updating
cc: @maleadt

### PR DESCRIPTION
**Pull request by DilumAluthge**
Fixes #12

### CODE REVIEW

**Comment by DilumAluthge** [comment]
@maleadt Looks like Travis is working again, for now, since the repo size is smaller (approx 32G).

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+    - master
+git:
+  depth: 1
 language: node_js
 node_js:
   - node
 os: linux
 
 script:
-  - npm install tools/pkgeval_badges
```
Comment (maleadt) [inline .travis.yml:10]: Why was this change needed?

**Comment by maleadt** [comment]
Thanks!

File: .travis.yml (hunk 1 above)
Comment (DilumAluthge) [reply .travis.yml:10]: Not sure, but the build wouldn't pass until I made those changes.

**Comment by DilumAluthge** [comment]
@maleadt Travis has basically killed its free tier (https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing), so we will need to move this build to GitHub Actions.
